include/overlay021/pokedexencdata.h

```diff
@@ -0,0 +1,155 @@
+#ifndef POKEPLATINUM_POKEDEXENCDATA_H
+#define POKEPLATINUM_POKEDEXENCDATA_H
+
+#include "constants/species.h"
+
+#include "cell_actor.h"
+
+typedef struct {
+    int *locations;
+    int numLocations;
+} EncounterLocations;
+
+typedef struct {
+    u8 x;
+    u8 y;
+    u8 isMtCoronet;
+    u8 unused;
+} DungeonCoordinates;
+
+typedef struct {
+    u8 y;
+    u8 x;
+    u8 height;
+    u8 width;
+    u8 cellMatrix[32];
+} FieldCoordinates;
+
+enum PokedexEncFileCatgegory {
+    PEFC_DUNGEONMORNING = 0,
+    PEFC_DUNGEONDAY = 1,
+    PEFC_DUNGEONNIGHT = 2,
+    PEFC_DUNGEONSPECIAL = 3,
+    PEFC_DUNGEONSPECIALNATDEX = 4,
+    PEFC_FIELDMORNING = 5,
+    PEFC_FIELDDAY = 6,
+    PEFC_FIELDNIGHT = 7,
+    PEFC_FIELDSPECIAL = 8,
+    PEFC_FIELDSPECIALNATDEX = 9,
```

Comment:
No need to explicitly label `enum` definitions like this past the first, unless there are jumps in the sequence. Typically, don't; let the compiler work for you.